logservice.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/urfave/cli"
+)
+
+var VERSION string
```

Comment:
To use this, we need to use LDFLAGS like this: https://github.com/screwdriver-cd/gitversion/blob/master/wercker.yml#L31